### ISSUE TITLE
[Consistency] User encounters a component or pattern that isn't considered a best practice.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#80603]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Benefit Tools
**Product:** Claim Status Tool
**Feature:** Improved Evidence Request Initiative

## Findings details
**VA.gov Experience Standard - issue:** User encounters a component or pattern that isn't considered a best practice.
**VA.gov Experience Standard - category:** Consistency
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @it-harrison (QA)

### Description

If I look at the Overview tab on the Details page of a claim, the process list does not use the Design System's va-process-list component

## Recommended action

Replace imposter process-list with va-process-list component

## References

- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Process list ](https://design.va.gov/components/process-list)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by skylerschain** [comment]
Just want to note for this one that we have a [big effort](https://github.com/department-of-veterans-affairs/va.gov-team/issues/71293) underway to totally redesign the "Overview" tab, so the reason we didn't update to the process list as part of this Epic is because that work will come soon after. So we will fix this issue as part of that much larger effort.

**Comment by jacobworrell** [comment]
This feedback is being incorporated into our Claim Phase Expansion work, which we have separate tickets for. Linking the epic to this issue and closing.

**Comment by shiragoodman** [comment]
sorry @jacobworrell - we ask that all Staging Review issue tickets remain open until the issue is resolved. Thank you for your understanding. I am reopening this ticket.

**Comment by pmclaren19** [comment]
This ticket is blocked by https://github.com/department-of-veterans-affairs/va.gov-team/issues/88061 .

**Comment by pmclaren19** [comment]
This has been resolved.
